README.md

```diff
@@ -6,11 +6,11 @@ Raise an [issue to join the EddieHub GitHub organisation](https://github.com/Edd
 
 ### Some vague idea of how we could turn this into a support channel.
 
-Imagine the scenario where you are helping a stranger (or a new friend), a friend or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported, a Node.js article that isn't supported in Node v14 and that sort of thing...
+Imagine the scenario where you are helping a stranger, a friend or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported or a Node.js article that isn't supported in Node v14. That sort of thing...
 
-You volunteer to help that person, you pair on it, it's fixed and they're super happy.
+You volunteer to help that person and pair on it. You fix it. They're super happy.
```

Comment:
I think this one's better?:
`You volunteer to help that person and pair on it. You fix it and that made them super happy.`

Reply:
>You volunteer to help that person and pair on it. You fix it and that made them super happy.

This looks good. But, how about changing "made" to "make" to maintain the present tense being used?

Reply:
Yeah better use "make"

make = present
made = past

Reply:
Making the change now